### PR TITLE
display prices on view and capture exceptions

### PR DESCRIPTION
**Pull request by LeeRenJie**
depends on #121

### CODE REVIEW

File: finance-tracker/app/models/Copilot [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Synthesizing 7/10 solutions (Duplicates hidden)
```
Comment (tlkiong) [inline finance-tracker/app/models/Copilot:1]: ??

File: finance-tracker/config/routes.rb [hunk 2]
```diff
@@ -2,4 +2,6 @@
   devise_for :users
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
   root 'welcome#index'
+  get 'my_portfolio', to: 'users#my_portfolio'
+  get 'search_stock', to: 'stocks#search'
```
Comment (tlkiong) [inline finance-tracker/config/routes.rb:6]: use `resources`

**Review by tlkiong — CHANGES_REQUESTED** [review]
.

File: finance-tracker/config/routes.rb (hunk 2 above)
Comment (LeeRenJie) [reply finance-tracker/config/routes.rb:6]: but then the actions are not index, delete, new, create etc wor, how to use resources with specific URL and action?

Comment (tlkiong) [reply finance-tracker/config/routes.rb:6]: ganbatte

**Review by tlkiong — CHANGES_REQUESTED** [review]
.

**Review by tlkiong — APPROVED** [review]

**Pull request merged**